src/components/OrganizationNameAndIcon.tsx

```diff
@@ -1,19 +1,22 @@
 import AccountBalanceIcon from '@mui/icons-material/AccountBalance';
 import { Box, BoxProps, Skeleton, Tooltip, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { CristinApiPath } from '../api/apiPaths';
 import { useFetchOrganization } from '../api/hooks/useFetchOrganization';
 import { getLanguageString } from '../utils/translation-helpers';
-import { useTranslation } from 'react-i18next';
 
 interface OrganizationNameAndIconProps extends Pick<BoxProps, 'sx'> {
   id: string;
-  acronym?: boolean;
 }
 
-export const OrganizationNameAndIcon = ({ id, acronym = false, sx }: OrganizationNameAndIconProps) => {
+export const OrganizationNameAndIcon = ({ id, sx }: OrganizationNameAndIconProps) => {
   const { t } = useTranslation();
-  const organizationQuery = useFetchOrganization(id);
+
+  const isValidOrgId = id && id.includes(CristinApiPath.Organization);
+  const organizationQuery = useFetchOrganization(isValidOrgId ? id : '');
   const orgName = getLanguageString(organizationQuery.data?.labels);
   const orgAcronym = organizationQuery.data?.acronym;
+  const displayName = isValidOrgId ? orgAcronym : id;
```

Comment:
Hva skjer om displayname defaulter til id her? Langt?

Reply:
Aner ikke hva som er reellt her da, men ser den blir litt rart om jeg hardkoder noe langt ja. Skal se litt på det
![bilde](https://github.com/user-attachments/assets/2954b191-c9bf-4181-a66e-9889c167c293)

Reply:
Merger as-is, så vi rekker utrulling før dagens møte, men ser videre på det!